SECURITY.md

```diff
@@ -0,0 +1,17 @@
+# Security Policy
+
+## Reporting a Vulnerability
+
+To report a vulnerability, please use GitHub's [vulnerability report](https://github.com/metriport/metriport/security) on the affected
+repository or email us at support@metriport.com
+
+If you believe you have found a security vulnerability in the Metriport API and/or source code, we encourage you to
+[let us know](https://github.com/metriport/metriport/security) right away.
+
+We will investigate all legitimate reports and do our best to quickly fix the problem.
+
+```
+- Contact: https://github.com/metriport/metriport/security or support@metriport.com
```

Comment:
We could have a dedicated email for this, although I think we want folks to use GH's vulnerability report instead of email, so it should be fine to have support@ as alternative?